gkeys/gkeys/base.py

```diff
@@ -227,6 +227,10 @@ def _option_email(parser=None):
         parser.add_argument('-E', '--email', dest='email', default=None,
             help='Email parameter for sending email reminders')
 
+    @staticmethod
+    def _option_user(parser=None):
+        parser.add_argument('-U', '--user', dest='user', default=None,
+            help='User parameter for email login')
```

Comment:
while email login is the only use so far... it could apply to other future actions.  Try to make the description a little more generic.  Just like I asked you to do for the email option.